### DISCUSSION TITLE
Retrieve data from the server and then update the app

### DISCUSSION THREAD

**Discussion by yanparent** [GetPublii/Publii#1697] [Help]
Hi everyone,

I wonder if it's possible to do a "reverse sync" by retrieving data directly on the server, and then update the local app based on this.

To add context to my query : I've used to update my publii website on a pro computer, which has been now returned back to the company...and I can't put the hand anymore on my local datas.

Thanks a lot for all your advices,

Yan

**Comment by michaelfluegge** [comment]
This was being asked several times before. If you do not have a backup made with Publii, it is not possible to import data from the server. Try searching the discussions to find some explanation about this.

**Reply by candidexmedia** [reply]
See: https://github.com/GetPublii/Publii/discussions/1515
